website/server/models/challenge.js

```diff
@@ -149,7 +154,7 @@ schema.methods.syncToUser = async function syncChallengeToUser (user) {
     }
   });
 
-  toSave.push(user.save());
+  toSave.push(User.collection.update({_id: user._id}, user.toObject()));
```

Comment:
From what @paglias said below he didn't sound convinced that this would actually fix the problem so I didn't know if I should keep going. I can make this change and update it if that's the way we want to go